### DISCUSSION TITLE
Which protocols can we use to transfer data?

### DISCUSSION THREAD

**Discussion by KazunaArai** [eclipse-edc/Connector#4132] [Q&A] [answered]
Hello, I am really new to use EDC connector and need help to understand its system. 
My question is which protocols (HTTP, FTP, NGSI, etc) we can use to transfer data from provider side to cosumer side. Are there any limitations?
I appreciate if you help me. Thank you in advance!

**Comment by paullatzelsperger** [comment, marked as answer]
don't know if this is still relevant, but EDC comes with modules for HTTP and Kafka, and you'll find CSP-specific implementations in the `Technology-*` repositories.

And of course, you can implement your own transfer facilities. For that, you'll need a data plane implementation, and possibly an provisioner implementation. I recommend you look at one of the `Technology-*` repositories to see how that is done.

- [Technology-Azure](https://github.com/eclipse-edc/Technology-Azure)
- [Technology-Aws](https://github.com/eclipse-edc/Technology-Aws)
- [Technology-Gcp](https://github.com/eclipse-edc/Technology-Gcp)

**Reply by KazunaArai** [reply]
i'll check those repositories, thanks!
